src/fs/StandardDirectory.cxx

```diff
@@ -127,7 +127,7 @@ static AllocatedPath GetStandardDir(int folder_id)
 
 #ifdef USE_XDG
 
-static const char home_prefix[] = "$HOME/";
+static constexpr auto home_prefix = "$HOME/";
```

Comment:
This is not a `std::array` conversion, is it?

Reply:
More like a C array conversion to const char*. IIRC clang-tidy warns on it.

Reply:
"some random tool warns" alone does not convince me. If you can't explain a change, please do not submit. And please do not submit changes which you did not test.

Reply:
It's gone now.